docs/document/content/features/dist-sql/syntax/rql/rql-readwrite-splitting-rule.en.md

```diff
@@ -11,22 +11,22 @@ SHOW READWRITE_SPLITTING RULES [FROM schemaName]
 
 ## Description
 
-| Column                  | Description                          |
-| ----------------------- | ------------------------------------ |
-| name                    | Rule name                            |
-| autoAwareDataSourceName | Auto-Aware discovery data source name（Display configuration dynamic read-write separation rules）|
-| writeDataSourceName     | Write data source name                |
-| readDataSourceNames     | Read data source name list            |
-| loadBalancerType        | Load balance algorithm type           |
-| loadBalancerProps       | Load balance algorithm parameter      |
+| Column                      | Description                          |
+| --------------------------- | ------------------------------------ |
+| name                        | Rule name                            |
+| auto_aware_data_source_name | Auto-Aware discovery data source name（Display configuration dynamic read-write separation rules）|
+| write_data_source_name      | Write data source name                |
+| read_data_source_names      | Read data source name list            |
+| load_balancer_type          | Load balance algorithm type           |
+| load_balancer_props         | Load balance algorithm parameter      |
 
 ## Example
 
 *Static Read-write Separation Rules*
```

Comment:
Thank you for your PR, can you change these ‘Read-write Separation’ to ‘Readwrite splitting’ by the way?

Reply:
> Thank you for your PR, can you change these ‘Read-write Separation’ to ‘Readwrite splitting’ by the way?

Done, please review, thank you.

Reply:
👍  It's OK now. Thank you!